img/dot-graphs/inclusion-rules-eval.dot

```diff
@@ -0,0 +1,28 @@
+// render with:
+// $ dot -Tpng inclusion-rules-eval.dot > inclusion-rules-eval.png
```

Comment:
@pcsengodi perhaps `_` instead of `-` would be preferable? Looks like I slipped into kebab-case for the dot files.

Reply:
For me both ways are fine.